docs/_writing-tests/general-guidelines.md

```diff
@@ -112,6 +112,32 @@ either installed or to have specific metrics. As such, in most cases
 when a known font is needed Ahem should be used. In other cases,
 `@font-face` should be used.
 
+#### Be Conservative
+
+Sometimes, when authoring tests for one web platform feature, it may be
```

Comment:
This seems to be duplicating what is under "Be Minimal" above. (Though "Be Conservative" is probably a better header!) We should probably add a paragraph there rather than this.

My suggestion would be something like:

> Tests should ensure they run correctly and fail when appropriate (including, e.g., when the feature is unsupported) on the latest stable release of Apple Safari, Google Chrome, Microsoft Edge, and Mozilla Firefox. They should, therefore, not rely on any features aside from the one under test unless they are supported in all four browsers.
>
> Patches to make tests run on older versions or other browsers will be accepted provided they are relatively simple and do not add undue complexity to the test.